jwst/undersampling_correction/undersampling_correction_step.py

```diff
@@ -0,0 +1,43 @@
+#! /usr/bin/env python
+import logging
+
+from ..stpipe import Step
+from .. import datamodels
+from . import undersampling_correction
+
+log = logging.getLogger(__name__)
+log.setLevel(logging.DEBUG)
+
+__all__ = ["UndersamplingCorrectionStep"]
+
+
+class UndersamplingCorrectionStep(Step):
+    """
+    This Step sets the UNDERSAMP flag for groups exhibiting significant
+    charge migration.
+    """
+    class_alias = "undersampling_correction"
+
+    spec = """
+        signal_threshold = float(default=30000)
+        skip = boolean(default=True)
+    """
+    def process(self, input):
+
+        # Open the input data model
+        with datamodels.RampModel(input) as input_model:
+            if (input_model.data.shape[1] < 3):  # skip step if only 1 or 2 groups/integration
+                log.warning('Too few groups per integration; skipping undersampling_correction')
```

Comment:
I'm not sure this should be a warning. Some users may interpret it as a bad thing. How about an `info` message. @bushouse?

Reply:
And can you add a test for this case - less than 3 groups.

Reply:
I agree. Change it from warning to just `info`.

Reply:
Done